### PR TITLE
Increases Nuclear Challenge Time Limit

### PR DESCRIPTION
**Pull request by Robertha89**
Time begins counting as soon as the server starts from 12:00, players spawn in at 12:03 followed by a small delay.

This doubles the time to 12:10 from 12:05 as a time limit to call the challenge. 

Less then 2 minutes of actual time is kinda little.

### CODE REVIEW

**Comment by Spacemanspark** [comment]
:+1:

**Comment by johannhawk** [comment]
i approve

**Comment by NTSAM** [comment]
:+1:

**Comment by Fox-McCloud** [comment]
changelog

**Comment by Plotron** [comment]
I approve. I want to see it in action more often.
When I observe and see nukies talking in the room, it's already past the time to decide about the challenge mode.

**Comment by Robertha89** [comment]
Changelog added.

File: html/changelogs/PPI-patch-1.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscdel: "Removes two minute limite for activate Nuclear Challenge as Nuke Ops Commander, after spawn."
+  - rscadd: "Adds a 7 minute time limit to activate the Nuclear Challenge as Nuke Ops Commander, after spawn."
```
Comment (Fox-McCloud) [inline html/changelogs/PPI-patch-1.yml]: This isn't adding or removing anything; this is a tweak.

**Comment by Fox-McCloud** [comment]
Why....are you putting merge pull requests into your PR when you're literally only changing your changelog.

**Comment by Robertha89** [comment]
Because i remembered you could do it 5 minutes after i did it the wrong way. I am not good at github.

**Pull request merged**